libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -16,14 +16,16 @@ import { PrevDirective } from './directives/prev.directive';
 import { NextDirective } from './directives/next.directive';
 
 import { HomeRoutingModule } from './home.routing';
+import { SocialImpactModule } from '@elewa-group/pages/elewa/social-impact'
 
 @NgModule({
   imports: [
     CommonModule,
     LayoutModule,
     NgOptimizedImage,
     CardsModule,
-    HomeRoutingModule
+    HomeRoutingModule,
+    SocialImpactModule
```

Comment:
Not allowed, page modules should not import each other. May cause circular dependency later on. Use shared libs.